hack/lokistack_gateway_dev.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: loki.openshift.io/v1beta1
+kind: LokiStack
+metadata:
+  name: lokistack-dev
+spec:
+  size: 1x.extra-small
+  replicationFactor: 1
+  storage:
+    secret:
+      name: test
+  storageClassName: standard
```

Comment:
```suggestion
  storageClassName: gp2
```

Reply:
Standard doesn't work for OCP. This is actually an error in the existing `hack/lokistack_dev.yaml`.